### PR TITLE
Error page

### PR DESCRIPTION
**Pull request by TorskyM**

### CODE REVIEW

File: src/pages/error-page/errorPage.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import React, { useEffect } from 'react';
```
Comment (Alan18081) [inline src/pages/error-page/errorPage.js:1]: Please, rename the file to the `error-page`, we name components using lowercase and dashes

File: src/pages/error-page/errorPage.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+  const classes = useStyles();
+  const dispatch = useDispatch();
+
+  const errorImagePath = isLightTheme
+    ? ERROR_PAGE_IMAGES.light
+    : ERROR_PAGE_IMAGES.dark;
+
+  useEffect(() => {
+    dispatch(getError());
```
Comment (Alan18081) [inline src/pages/error-page/errorPage.js:28]: What does this action actually do?

File: src/pages/error-page/errorPage.styles.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import { makeStyles } from '@material-ui/core/styles';
```
Comment (Alan18081) [inline src/pages/error-page/errorPage.styles.js:1]: Please, rename the file to the `error-page.styles.js`

File: src/pages/error-page/errorPage.js (hunk 1 above)
Comment (TorskyM) [reply src/pages/error-page/errorPage.js:1]: Done

File: src/pages/error-page/errorPage.styles.js (hunk 3 above)
Comment (TorskyM) [reply src/pages/error-page/errorPage.styles.js:1]: Done

File: src/pages/error-page/errorPage.js (hunk 2 above)
Comment (TorskyM) [reply src/pages/error-page/errorPage.js:28]: I changed the logic, it is no longer needed there

File: src/configs/index.js [hunk 4]
```diff
@@ -253,3 +258,14 @@ export const CONFIRM_ERROR = [
     value: 'Oops! Something went wrong.'
   }
 ];
+
+export const ERROR_PAGE_MESSAGE = [
```
Comment (Alan18081) [inline src/configs/index.js:262]: Please, move it from config to the `translations` folder, in a separate file. Look how other translations are implemented

File: src/pages/error-page/error-page.js [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+  const classes = useStyles();
+
+  const errorImagePath = isLightTheme
+    ? ERROR_PAGE_IMAGES.light
+    : ERROR_PAGE_IMAGES.dark;
+
+  return (
+    <div className={classes.wrapper}>
+      <div className={classes.error}>
+        <img className={classes.errorImage} src={errorImagePath} alt='Oops' />
```
Comment (Alan18081) [inline src/pages/error-page/error-page.js:28]: I think `alt` should be more meaningful and translated too

File: src/store/store.js [hunk 6]
```diff
@@ -12,9 +12,7 @@ const configureStore = () => {
   const store = createStore(
     rootReducer(history),
     compose(
-      applyMiddleware(routerMiddleware(history), sagaMiddleware),
-      window.__REDUX_DEVTOOLS_EXTENSION__ &&
```
Comment (Alan18081) [inline src/store/store.js:16]: Revert this change because we need `redux-devtools` in development environment

File: src/routes/routes.js [hunk 7]
```diff
@@ -33,6 +34,7 @@ const Routes = () => {
             exact
             render={({ match }) => <Confirmation token={match.params.token} />}
           />
+          <Route path='/error-page' exact component={ErrorRage} />
```
Comment (VolodymyrTrach) [inline src/routes/routes.js:37]: Why Rage? Please rename it to ErrorPage

File: src/routes/routes.js [hunk 8]
```diff
@@ -13,6 +13,7 @@ import AppFooter from '../components/app-footer';
 import Register from '../pages/register';
 import Login from '../pages/login';
 import Confirmation from '../pages/confirmation';
+import ErrorRage from '../pages/error-page';
```
Comment (VolodymyrTrach) [inline src/routes/routes.js:16]: Why Rage? Please rename it to ErrorPage

File: src/pages/error-page/error-page.js [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+const ErrorPage = () => {
+  const { language, isLightTheme, errorMessage } = useSelector(
+    ({ Language, Theme, Error }) => ({
+      language: Language.language,
+      isLightTheme: Theme.lightMode,
+      errorMessage: Error.error
+    })
+  );
+
+  const classes = useStyles();
```
Comment (DmytroDidukh) [inline src/pages/error-page/error-page.js:20]: please rename this variable to `styles` to match the project style
